.github/actions/build/action.yaml

```diff
@@ -65,7 +65,7 @@ runs:
         run: |
           set -x
           cd src
-          tar cvf - out/${{ matrix.platform }}_${{ matrix.config }}/*tests out/${{ matrix.platform }}_${{ matrix.config }}/*.so out/${{ matrix.platform }}_${{ matrix.config }}/*.so.* out/${{ matrix.platform }}_${{ matrix.config }}/starboard/*.so.* | xz -T0 -1 -z - > test_artifacts.tar.xz
+          tar cvf - out/${{ matrix.platform }}_${{ matrix.config }}/*tests out/${{ matrix.platform }}_${{ matrix.config }}/nplb out/${{ matrix.platform }}_${{ matrix.config }}/*.so out/${{ matrix.platform }}_${{ matrix.config }}/*.so.* out/${{ matrix.platform }}_${{ matrix.config }}/starboard/*.so.* out/${{ matrix.platform }}_${{ matrix.config }}/scontent/test | xz -T0 -1 -z - > test_artifacts.tar.xz
```

Comment:
I think this is fine to get nplb in but @oxve the infra team should figure out how to maintain this long-term (or let us know if you have an immediate solution)